manim/mobject/coordinate_systems.py

```diff
@@ -92,28 +92,31 @@ def get_axis_labels(self, x_label_tex="x", y_label_tex="y"):
         return self.axis_labels
 
     def get_graph(self, function, **kwargs):
-        x_min = kwargs.pop("x_min", self.x_min)
-        x_max = kwargs.pop("x_max", self.x_max)
+        x_min = kwargs.pop("x_min", kwargs.pop("t_min",self.x_min))
+        x_max = kwargs.pop("x_max", kwargs.pop("t_max",self.x_max))
```

Comment:
Question: if I'm reading this correctly, this allows the `get_graph` method to receive t_min instead of x_min. I'm on the fence about this :thinking:  In what instance is this necessary or desirable?

Reply:
I think it allows both x_min or t_min. I do agree it is quite unreadable though, I'll change that.
If I called get_graph(func,t_min=-2,t_max=2)
It'll run the calculation from right to left, and since there is a t_min, the inner `kwargs.pop(t_min,self.x_min)` will result in `t_min`.
Since there is no x_min, it'll just return the default which is `kwargs.pop(t_min,self.x_min)` which is `t_min`.
I wanted to have it in such a way that you could still use t_min and t_max AND x_min and x_max, since in a explicit graph they are one and the same. It's a cheap way of doing it that is hard to understand.

Reply:
Thanks. I do understand the code. My hesitation comes from the fact that this function can be called as `get_graph(f, t_min=.., t_max=.., x_min=.., x_max=..)` and only the `x_*` values would be used. Further, since this function doesn't have good documentation, it would be hard for anyone to realize that both x_* and t_* are possible.

I would strongly recommend using only `x_*` arguments or `t_*` arguments but not both, to avoid confusion and to simplify the code.